README.md

```diff
@@ -1,6 +1,6 @@
 # PoshUI Components
 
-### This repo contains the implementation of components of the [Posh UI](https://github.com/hsnice16/PoshUI-Documentation) component library.
+### This repo contains the implementation of components of the <a href="https://github.com/hsnice16/PoshUI-Documentation" target="_blank">Posh UI</a> component library.
```

Comment:
I guess the former line is the right way to add links in markdown. It's fine if the latter one is working.

Reply:
Thanks for pointing this, Sainath. I just got to know that `target="_blank"` does not work in github markdown. So I will change that.